### DISCUSSION TITLE
Trouble implementing OIDC authentication via Okta

### DISCUSSION THREAD

**Discussion by ivo-sudo** [aws-solutions/centralized-logging-with-opensearch#251] [Q&A]
Hi, I've deployed the stack successfully and I am able to navigate to the sign in URL but when I click on the orange button to login via Okta (my chosen IdP) I get the following message: "Oops... Client authentication failed. Either the client or the client credentials are invalid." What should the OIDC settings look like on the Okta side as I assume the problem is there?

**Comment by wchaws** [comment]
Hi @ivo-sudo, have you checked https://docs.aws.amazon.com/solutions/latest/centralized-logging-with-opensearch/launch-with-openid-connect-oidc.html#deployment-overview-1 this doc? You can refer other Idp configuration as an example. If this can not solve your problem. Can you please open browser's dev tools and check if there's any 4xx HTTP response?
